### DISCUSSION TITLE
How did you come across this game?

### DISCUSSION THREAD

**Discussion by Turtlecodez** [nroutasuo/level13#295] [General]
I'll go first.
A few years ago, I played the game "A Dark Room." It was really short, and I replayed it over and over because it was a great game. I was searching for weeks to try and find a game similar to ADR, and I wasn't having any luck. Then I came across this game, and I immediately fell in love with its simplistic yet amazing style. I've never actually ended up completing it yet, as I got reset as the updates came, but I still love it and am trying again.

**Comment by demoncoyote80** [comment]
I found it the same way, actually, lol. I found ADR while trying to find games to play in school.

**Reply by Turtlecodez** [reply]
both games are wonderful :) haven't played level 13 in a while because it's so long and i've lost my progress many times
